packages/emmett-esdb/tsconfig.eslint.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Those dummy files are needed for vscode eslint plugin, as it runs eslint config locally for packages. And as in the file, we have a reference for `tsconfig.eslint.json`. Then if it won't find such a file in the local package folder, then it will cry out loud and doesn't highlight eslint errors.